substrate/frame/recovery/src/benchmarking.rs

```diff
@@ -51,10 +50,7 @@ fn generate_friends<T: Config>(num: u32) -> Vec<<T as frame_system::Config>::Acc
 
 	for friend in 0..friends.len() {
 		// Top up accounts of friends
-		T::Currency::make_free_balance_be(
-			&friends.get(friend).unwrap(),
-			BalanceOf::<T>::max_value(),
-		);
+		T::Currency::set_balance(&friends.get(friend).unwrap(), TOP_UP.into());
```

Comment:
Using `BalanceOf::<T>::max_value()` with `set_balance` would quickly overflow, that's why we use a const way lower than that amount instead

Reply:
Another alternative is using max but shifting it down a few but either seems fine to me.